main.go

```diff
@@ -37,8 +37,7 @@ func init() {
 	base64Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/="
 	hexChars = "1234567890abcdefABCDEF"
 
-	stopWords = []string{"setting", "Setting", "SETTING", "info",
```

Comment:
Looks like you removed the Upper case cases so we would be calling `ToLower` on already lowered case strings. A better way to implement this would be to check this would be doing some case insensitive regex checks.

Reply:
We're calling ToLower on what we check the stopwords against, not the stopwords themselves.

Reply:
I've been staring at a screen too long. >.<
👍

Reply:
We all have days like that lol